presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/DatabaseShardManager.java

```diff
@@ -522,15 +522,15 @@ public ShardMetadata getShard(UUID shardUuid)
     }
 
     @Override
-    public ResultIterator<BucketShards> getShardNodes(long tableId, TupleDomain<RaptorColumnHandle> effectivePredicate)
+    public ResultIterator<BucketShards> getShardNodes(long tableId, boolean tableSupportsDeltaDelete, TupleDomain<RaptorColumnHandle> effectivePredicate)
```

Comment:
move `boolean tableSupportsDeltaDelete` to the end

Reply:
Same reason as above.